components/brave_new_tab_ui/components/default/binance/index.tsx

```diff
@@ -137,6 +137,7 @@ interface Props {
   disconnectInProgress: boolean
   authInvalid: boolean
   selectedView: string
+  stackPosition: number
```

Comment:
nit: instead of all this repeated props, you could have something like
```
[another file]
type WidgetProps {
   stackPosition: number
   ...
}

[this file]
type Props = WidgetProps & {
  ...binance-specific-props...
}
```